File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        if(len(nums)==0):
            return nums
        sum= nums[0]
        for i in range(1, len(nums)):
            nums[i] += sum
            sum+= nums[i] - sum
        return nums

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
//     TESESEASESE
    vector<int> runningSum(vector<int>& nums) {
        int sumSoFar = nums[0];
        for(int i=1; i< nums.size(); i++){
            nums[i] += sumSoFar;
            sumSoFar = nums[i];
        }
        return nums;
    }
};